src/UI/Art/SpriteArtist/SpriteLibrary.java

```diff
@@ -319,27 +317,10 @@ public static UnitSpriteSet getMapUnitSpriteSet(String type, Faction faction, Co
 
   private static void createMapUnitSpriteSet(UnitSpriteSetKey key)
   {
-    Faction faction = key.factionKey;
-
-    Function<AnimState,String> fileFinder = (state) -> getMapUnitSpriteFilename(key.unitTypeKey, faction.name, state);
-    if (!new File(getMapUnitSpriteFilename(key.unitTypeKey, faction.name)).canRead())
-      fileFinder = (state) -> getMapUnitSpriteFilename(key.unitTypeKey, faction.basis, state);
```

Comment:
We still need this logic, and you killed it.
I will say I've been thinking maybe it should try Thorn if the `basis` check fails, so we can have double-descent instead of single-descent. (e.g. Army has all DoR sprites, Wolves has only infantry and points to Army, and gets still-missing stuff from Thorn)

Reply:
So we do, and so I did. Should be fixed.